contrib/scrape-ec2-prices.py

```diff
@@ -101,12 +110,15 @@
 REGION_NAME_MAP = {
     'us-east': 'ec2_us_east',
     'us-east-1': 'ec2_us_east',
+    'us-east-2': 'ec2_us_east',
     'us-west': 'ec2_us_west',
     'us-west-1': 'ec2_us_west',
     'us-west-2': 'ec2_us_west_oregon',
     'eu-west-1': 'ec2_eu_west',
+    'eu-west-2': 'ec2_eu_west',
```

Comment:
If I'm not mistaken, this region is yet to be defined in the driver. That being said, it's worth having but the value should probably be different from the one mapped to by `eu-west-1`. How `ec2_eu_west_2` instead?